Directory.Build.props

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <eBPFExtensionsVersionMajor>0</eBPFExtensionsVersionMajor>
-    <eBPFExtensionsVersionMinor>1</eBPFExtensionsVersionMinor>
```

Comment:
why not only have the script in this PR; and follow up with a second PR generated by the script to bump up the version?

Reply:
moved update script to another pr